sdcm/cluster.py

```diff
@@ -1872,10 +1874,73 @@ def run_nodetool(self, sub_cmd, args="", options="", ignore_status=False):
         """
         cmd = self._gen_nodetool_cmd(sub_cmd, args, options)
         result = self.remoter.run(cmd, ignore_status=ignore_status)
-        self.log.debug("Command '%s' duration -> %s s", result.command,
-                       result.duration)
+        self.log.debug("Command '%s' duration -> %s s" % (result.command, result.duration))
+
         return result
 
+    def check_node_health(self, target_node=None):
+        self.check_nodes_status()
+        self.check_schema_version()
+
+    def check_nodes_status(self, target_node=None):
+        # Validate nodes health
+        # TODO: improve this part (using get_nodetool_status function from ScyllaCluster)
+        ClusterHealthValidatorEvent(type='start', name='NodesStatus', status=True, node=self.name)
+        try:
+            if not target_node:
```

Comment:
I don't understand why do we need to classify nodes

Reply:
I just saved the current procedure

Reply:
OK, Is there an empty node type? As far as I understand there are two types, regular and the one that Nemesis runs on.
Use `running_nemesis` attribute to decide if the node is "target" instead of passing `target_node` everywhere and remove the "empty" node type

Reply:
Fixed